contrib/cloud.google.com/go/pubsub.v1/pubsub_test.go

```diff
@@ -0,0 +1,219 @@
+package pubsub
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	"cloud.google.com/go/pubsub/pstest"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc"
```

Comment:
This group goes last, separated by an empty line from the `gopkg.in/DataDog/dd-trace-go.v1/...` imports.